pydruid/client.py

```diff
@@ -386,11 +403,20 @@ class PyDruid(BaseDruidClient):
     def __init__(self, url, endpoint):
         super(PyDruid, self).__init__(url, endpoint)
 
+    def ssl_context(self):
```

Comment:
This should at least be called something like `insecure_ssl_context()`. The current name is misleading, possibly dangerously so since a caller might not realize the context is insecure.

I think, though, that it'd be even better to have `ssl_context()` always get called by `_post` and for it to do something different based on the configuration. Then it would make sense to keep the current name.